jobs/atc/spec

```diff
@@ -507,3 +507,38 @@ properties:
     description: |
       Client secret for CredHub authorization.
     default: ""
+
+  aws-ssm.aws_region:
+    description: |
+      AWS region to use for fetching SSM parameters. Enviroment variable AWS_REGION is used
```

Comment:
I appreciate that these docs are somewhere but IMO the BOSH release is the wrong place. With BOSH, a user will only ever be setting properties in the manifest, never configuring environment variables or setting up `~/.aws` on the VMs or passing flags to CLIs - properties are a higher level of abstraction, where the user doesn't have to care how they work.